ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/storage/keyvalue/KeyValueStorageWorldStateStorage.java

```diff
@@ -29,23 +32,46 @@ public KeyValueStorageWorldStateStorage(final KeyValueStorage keyValueStorage) {
   }
 
   @Override
-  public Optional<BytesValue> getCode(final Hash codeHash) {
-    return keyValueStorage.get(codeHash);
+  public Optional<BytesValue> getCode(final Bytes32 codeHash) {
```

Comment:
Some of these methods use `Hash` and some use `Bytes32` - just changed them all to `Bytes32` since that is more generic.

Reply:
It's also kind of annoying to have to wrap `Bytes32` in `Hash` all over.  Kind of wondering if we really need the `Hash` type ...

Reply:
I like `Hash` providing a very clear type to say this is a hash, not just an arbitrary 32 bytes but I don't think it has to be a hard and fast "every hash must use the `Hash` type" kind of thing.  Particularly in this low level kind of place where the meaning is unambiguous using `Bytes32` makes sense to me.